config/app.php

```diff
@@ -1144,21 +1144,21 @@
     | Timezone for date displays
     | Defaults to UTC
     */
-    // 'default_setting_timezone' => env('DEFAULT_SETTING_TIMEZONE', 'UTC'),
+    'default_setting_timezone' => env('DEFAULT_SETTING_TIMEZONE', 'UTC'),
```

Comment:
I wonder why are all the default settings commented here - they are coming from .env and stock .env.example should have the correct values. I think we should uncomment all the defaults here and let the user choose defaults either in .env or configure per-user site settings.

Reply:
Copy that, can i add them via this PR: https://github.com/cypht-org/cypht/pull/1158 ?

Reply:
Sure, thanks!